CMakeLists.txt

```diff
@@ -36,6 +41,11 @@ target_link_libraries(${CJSON_UTILS_LIB} ${CJSON_LIB})
 install(TARGETS ${CJSON_UTILS_LIB} DESTINATION lib${LIB_SUFFIX})
 install(FILES cJSON_Utils.h DESTINATION include/cjson)
 
+set_target_properties(${CJSON_UTILS_LIB}
+    PROPERTIES
+        SOVERSION ${CJSON_VERSION_SO}
```

Comment:
Soversions should only be changed when the ABI changes and for the two libraries this might not be at the same time. Therefore the version of the two libs might have different version at some point. Maybe two version variables now might sense.